libs/util/common/src/lib/date.ts

```diff
@@ -24,3 +24,36 @@ export function formatDateAgo(date: Date | string | number) {
 		locale: de
 	});
 }
+
+export function formatMillisecondToString(ms: number): string {
+	const totalMinutes = Math.floor(ms / 60000);
+	const totalHours = Math.floor(totalMinutes / 60);
+	const days = Math.floor(totalHours / 24);
+	const hours = totalHours % 24;
+	const minutes = totalMinutes % 60;
+
+	let result = "";
+
+	if (days > 0) {
+		result += `${days} ${days > 1 ? "Tage" : "Tag"}`;
+	}
+
+	if (hours > 0) {
+		if (result) result += " und ";
+		result += `${hours} ${hours > 1 ? "Stunden" : "Stunde"}`;
+	}
+
+	if (minutes > 0) {
+		if (result) result += " und ";
+		result += `${minutes} ${minutes > 1 ? "Minuten" : "Minute"}`;
+	}
+
+	return result || "0 Minuten";
+}
```

Comment:
methode mit kommentar versehen für bsp ausgabe. 

also zB input: 1722348770174
output: 19934 Tage und 14 Stunden und 12 Minuten.

eine größere wiederverwendung würde entstehen, wenn statt ms, direkt ein date objekt übergeben werden würde. 

beim aufruf wäre dann folgendes notwendig format(newDate(ms));

dann sollte sich der methodenname aber ausreichend doll von der formatDateToString abgrenzen

und dann noch testfall schreiben.

Reply:
"Aber Dates sind doch Zeitpunkte und keine Intervalle. Dann müsste man 2 Dates benutzen und dann die Differenz bilden. Ich glaube, der Name ist einfach nicht gut. Die Methode wandelt ein Zeitintervall in Millisekunden in einen String um, der die Zeit in Stunden etc. angibt. Das bedeutet, wenn ich dem ein Intervall von 6 Stunden gebe, kommt da 6 Stunden nach Unix Time Start als Datum heraus. Klar, ich kann die Millisekunden-Zahl dann trotzdem wieder aus dem Objekt nehmen, aber das ist dann ja nur hin und her. Und das kann auch zu Verwirrung führen. Den Test mache ich fertig.